js/vendor/owncloud/share.js

```diff
@@ -256,19 +256,18 @@
 					 t('core', 'Allow editing') + '</label>';
 					 html += '</div>';
 					 }
-					 html += '</div>';
-					 var mailPublicNotificationEnabled = $(
-					 'input:hidden[name=mailPublicNotificationEnabled]').val();
-					 if (mailPublicNotificationEnabled === 'yes') {
-					 html += '<form id="emailPrivateLink">';
-					 html +=
-					 '<input id="email" style="display:none; width:62%;" value="" placeholder="' +
-					 t('core', 'Email link to person') + '" type="text" />';
-					 html +=
-					 '<input id="emailButton" style="display:none;" type="submit" value="' +
-					 t('core', 'Send') + '" />';
-					 html += '</form>';
-					 }*/
+					 html += '</div>';*/
```

Comment:
Actually, I was also thinking of removing that line but as in this PR it was in the commented out code, I didn't remove it. I have removed it in my PR for #565, which I am about to make.